tensorflow/core/api_def/base_api/api_def_Diag.pbtxt

```diff
@@ -20,10 +20,11 @@ For example:
 
 ```
 # 'diagonal' is [1, 2, 3, 4]
-tf.diag(diagonal) ==> [[1, 0, 0, 0]
-                       [0, 2, 0, 0]
-                       [0, 0, 3, 0]
-                       [0, 0, 0, 4]]
+tf.linarg.tensor_diag(diagonal) ==> <tf.Tensor: shape=(4, 4), dtype=int32, numpy=
```

Comment:
The examples should be docstests as per https://www.tensorflow.org/community/contribute/docs_ref